src/main/java/com/dev/cinema/model/dto/hall/CinemaHallRequestDto.java

```diff
@@ -1,9 +1,11 @@
 package com.dev.cinema.model.dto.hall;
 
+import javax.validation.constraints.NotNull;
 import lombok.Data;
 
 @Data
 public class CinemaHallRequestDto {
+    @NotNull
```

Comment:
You should add message in bracket (message = "capacity shouldn't be null")

Reply:
done